lib/sync_checker/formats/edition_base.rb

```diff
@@ -51,7 +51,7 @@ def base_paths
       end
 
       def get_path(locale)
-        path = "#{root_path}#{document.slug}"
+        path = Whitehall::UrlMaker.new.public_document_path(edition_expected_in_live)
```

Comment:
I think we might need to use `edition_expected_in_draft`. I think the live one will be `nil` for new drafts. `_in_draft` should always be present. Well... it should in a fair and rational world which we're obviously not in here.